files/image_config/config-setup/config-setup

```diff
@@ -171,33 +174,6 @@ ztp_is_enabled()
     return $rv
 }
 
-# Load requested SONiC configuration into config DB and initialize it
-# Usage: load_config <config_file>
-#
-#
-load_config()
-{
-    CONFIG_FILE=${1}
-    if [ "${CONFIG_FILE}" = "" ]; then
-        return 1
-    fi
-
-    sonic-db-cli CONFIG_DB FLUSHDB
-    sonic-cfggen -j ${CONFIG_FILE} --write-to-db
-    if [ $? -ne 0 ]; then
-        return $?
-    fi
-
-    if [[ -x /usr/local/bin/db_migrator.py ]]; then
-        # Migrate the DB to the latest schema version if needed
-        /usr/local/bin/db_migrator.py -o migrate
-    fi
-
-    sonic-db-cli CONFIG_DB SET "CONFIG_DB_INITIALIZED" "1"
```

Comment:
I think we have a dependency on this CONFIG_DB_INITIALIZED elsewhere. Do we not need this?

Reply:
@prsunny this is done here: https://github.com/Azure/sonic-utilities/blob/master/config/main.py#L1106